src/js/components/search/SearchOption.jsx

```diff
@@ -47,41 +48,63 @@ export default class SearchOption extends React.Component {
 
     render() {
         let searchOption = null;
+        let statusClass = '';
+        let comingSoon = (
+                <div>
+                    <div className="coming-soon-icon">
+                        <Icons.ExclamationCircle />
+                    </div>
+                    <span className="coming-soon-label">Coming Soon</span>
+                </div>
+            );
         switch (this.props.name) {
             case 'Search':
                 searchOption = (<SearchBox />);
+                statusClass = 'coming-soon';
                 break;
             case 'Award Type':
                 searchOption = (<AwardTypeContainer />);
+                statusClass = '';
+                comingSoon = '';
                 break;
             case 'Time Period':
                 searchOption = (<TimePeriodContainer />);
+                statusClass = '';
+                comingSoon = '';
                 break;
             case 'Agencies':
                 searchOption = (<AgencyContainer />);
+                statusClass = '';
+                comingSoon = '';
                 break;
             case 'Place of Performance':
                 searchOption = (<LocationSearchContainer />);
+                statusClass = '';
+                comingSoon = '';
                 break;
             case 'Recipients':
                 searchOption = (<RecipientSearchContainer />);
+                statusClass = '';
+                comingSoon = '';
                 break;
             default:
-                searchOption = null;
+                searchOption = comingSoon;
```

Comment:
I agree. Set it to null in line 52, set it to the JSX component in the default block in line 90.